providers/tests/cncf/kubernetes/test_pod_generator.py

```diff
@@ -725,10 +642,7 @@ def test_deserialize_non_existent_model_file(self, caplog, tmp_path):
         ),
     )
     def test_pod_name_confirm_to_max_length(self, input):
-        with pytest.warns(
-            AirflowProviderDeprecationWarning, match="Use `add_pod_suffix` in `kubernetes_helper_functions`"
-        ):
-            actual = PodGenerator.make_unique_pod_id(input)
+        actual = add_unique_suffix(name=input, max_len=100)
```

Comment:
the default `max_len` in `add_unique_suffix`  is 63 but when looking at `make_unique_pod_id` the default is 100, I fix the test accordingly but wonder if there is other solution

Reply:
cc @jedcunningham WDYT?

Reply:
I don't see a big problem here, shorter should be fine. I don't view changing pod names as a breaking change.

And yeah, probably better to refactor this test with the new limit.